src/main/java/org/dcsa/ovs/controller/VesselController.java

```diff
@@ -24,6 +25,12 @@ public VesselService getService() {
         return vesselService;
     }
 
+    @Override
+    @GetMapping(value = "{vesselIMONumber}")
+    @ResponseStatus(HttpStatus.OK)
+        public Mono<Vessel> findById(@PathVariable String vesselIMONumber) {
+        return getService().findById(vesselIMONumber);
```

Comment:
Should we test if it is a valid VesselIMONumber - no reason to try to access database if the IMO is invalid (as we only store vessels with valid IMO numbers)

Reply:
Done in https://github.com/dcsaorg/DCSA-Event-Core/pull/46